server/src/main/java/keywhiz/service/daos/SecretSeriesDAO.java

```diff
@@ -80,6 +80,13 @@ long createSecretSeries(String name, String creator, String description, @Nullab
     return r.getId();
   }
 
+  public int setCurrent(long secretId, long secretContentId) {
```

Comment:
It will imply an extra round trip to the database but seems like a good thing to have.